lib/list.js

```diff
@@ -41,30 +45,30 @@ function validate (opts) {
   if (opts.age && !R.contains(opts.age, R.values(c.age))) {
     throw Error(`Invalid age range ${opts.age}`);
   }
+}
+
+function defineCollectionUrl (opts) {
+  const { collection, category } = opts;
+  const path = c.paths[collection];
 
-  opts.lang = opts.lang || 'en';
-  opts.country = opts.country || 'us';
-  opts.num = opts.num || 500;
+  return (category)
```

Comment:
It's very hard to follow what url is being generated here, and how does it relate to the objects in the constants module.